konsent/__init__.py

```diff
@@ -301,7 +301,7 @@ def register_union():
         msg = 'Your union is now registered and can be accessed by other users'
         return render_template('index.html', msg=msg)
     return render_template('register-union.html',
-        form=form, unions=print_unions())
+        form=form, unions=Union.print())
```

Comment:
`print` is a reserved keyword. Name it something else to avoid a SyntaxError.

Reply:
`print` is a builtin function in python3 so this is perfectly fine, are you sure you didn't run it with python2?

Reply:
we should probably make these functions one tbh, printing and listing unions it's kinda the same thing.

Reply:
Oh fuck, sorry, now I see what you're trying to do. I was running with 2.7. I'll test again with 3.